engine/execution/computation/computer/computer.go

```diff
@@ -407,19 +405,18 @@ func (e *blockComputer) executeTransaction(
 	res.AddTransactionResult(&txResult)
 	res.AddComputationUsed(tx.ComputationUsed)
 
+	runtime.ReadMemStats(&m)
+	memAllocAfter := m.TotalAlloc
+
 	evt := e.log.With().
 		Hex("tx_id", txResult.TransactionID[:]).
 		Str("block_id", res.ExecutableBlock.ID().String()).
 		Str("traceID", traceID).
 		Uint64("computation_used", txResult.ComputationUsed).
+		Uint64("memory_used", tx.MemoryUsed).
+		Uint64("memAlloc", memAllocAfter-memAllocBefore).
```

Comment:
not something introduced in this PR, and probably not a big deal: wouldn't it be nice to use snake-case for all names for consistency? e.g: `memory_alloc`